lib/utils/unbox.js

```diff
@@ -35,19 +35,16 @@ function verifyURL(url) {
 
     var options = {
       method: 'HEAD',
-      host: configURL.host,
-      path: configURL.path
+      uri: 'https://' + configURL.host + configURL.path
     };
-    var req = https.request(options, function(r) {
+    request(options, function(error, r) {
```

Comment:
@micahriggan actually one comment - would you mind adding a check for `error` now that it's a new arg?

Reply:
```
~       if (error) {
+         return reject(new Error("Error making request. Please check the format of the requested resource: " + options.uri));
+       } else if (r.statusCode == 404) {
```

Would something like this work?